.github/workflows/build-test-images.yml

```diff
@@ -143,6 +143,11 @@ jobs:
           service_account_key: ${{ secrets.GCP_SA_KEY }}
           export_default_credentials: true
 
+      - name: Authenticate to ghcr.io
```

Comment:
Wondering; would it be better to rewrite this workflow to use the `docker-build-push` action? https://docs.docker.com/ci-cd/github-actions/

(I could ask @crazy-max if he's able to assist)

Reply:
For now, let's at least use the login-action for this: https://github.com/docker/login-action#github-container-registry:
```
        name: Login to GitHub Container Registry
        uses: docker/login-action@v1
        with:
          registry: ghcr.io
          username: ${{ github.actor }}
          password: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
Ah, yes, good suggestion; that's a good step. (I just started to look at the workflow to see what it's all doing, and it looks somewhat complex, so may need some thinking how to change it)

Reply:
> 

Ohh, that is so much better. Didn't even think to search for an action for this. Good thing to remember. Thanks Phil!

Reply:
Yes that might be easier, lgtm

Reply:
The same could be done for `Azure login` step: https://github.com/docker/login-action#azure-container-registry-acr

Reply:
@thaJeztah  The workflow is indeed a bit complex. It leverages a Windows helper VM to build windows images. It also creates a manifest and includes OS versions for windows images. On Windows, the version of the container needs to match the version of the host. If Hyper-V isolation is used, you can run containers built for older versions of Windows, but not newer. This is why the workflow uses the Azure CLI action to spawn a VM in Azure (Windows Server 2022), and not a separate job. The latest github windows action runner seems to be Windows server 2019, which means we would only be able to build container images up to that version.

If the ```docker-build-push``` action allows us to leverage a remote helper, I think we could rewrite it, but it would take a while.